### PR TITLE
Highlight that CUJs need to be written in the TDD doc and the doc must be linked to the feature request form

### PR DESCRIPTION
**Pull request by kevintab95**
Highlight that CUJs need to be written in the TDD doc and the doc must be linked to the feature request form

### CODE REVIEW

File: Launching-new-features.md [hunk 1]
```diff
@@ -3,7 +3,8 @@ Please use the following process for launching major new features that lead to a
 - Before doing anything else, please:
     - Make sure e2e tests are present for your feature.
     - Make sure it is easy to toggle on and off, in case it needs to be aborted for the release. This implies proper gating of views, controllers, etc.
-- Fill out this [google form](https://forms.gle/m2u1VkQDXWee4euAA).
+    - Make sure that there is a clear set of CUJs for the feature listed in the [TDD](https://docs.google.com/document/d/1mnz8f708DZIa6BpUyRmbb0gCT6EKO3wIvWa_3rOEOYs/edit#bookmark=id.c1jeenukq729) doc for the feature.
```
Comment (seanlip) [inline Launching-new-features.md]: Maybe explain which part of the TDD to look at (state the name of the section)

File: Launching-new-features.md [hunk 2]
```diff
@@ -3,7 +3,8 @@ Please use the following process for launching major new features that lead to a
...
+- Fill out this [google form](https://forms.gle/m2u1VkQDXWee4euAA). Please provide a link to the TDD in the form.
 - Nithesh (**@nithusha21**) will first do a quick audit of the feature, in order to ensure that it is ready for testing. If the feature is not ready, he will get back to you with what needs to be done, and abort the launch. If the feature passes initial checks, he will put you in touch with a product tester. (The expected turnaround time for this part is < 48 hours.)
```
Comment (seanlip) [inline Launching-new-features.md]: We should probably change this. Would you like to do it, or should we discuss with @vojtechjelinek?

**Review by seanlip — CHANGES_REQUESTED** [review]
Thanks @kevintab95! Left a couple comments.

File: Launching-new-features.md [hunk 3]
```diff
@@ -3,7 +3,8 @@ Please use the following process for launching major new features that lead to a
 - Before doing anything else, please:
     - Make sure e2e tests are present for your feature.
     - Make sure it is easy to toggle on and off, in case it needs to be aborted for the release. This implies proper gating of views, controllers, etc.
```
Comment (seanlip) [inline Launching-new-features.md]: Should we be more prescriptive here about using dynamic feature gating? /cc @nithinrdy

Comment (nithinrdy) [reply Launching-new-features.md]: Hey, sorry for the delay.

a. Yes, perhaps we could add a link to the feature gating wiki page, and
b. We could add a note right after which talks about writing E2E tests for gated features, something like this:

`To write E2E tests for features gated behind a feature flag, you may need to use the 'enableFeatureForProd'/'enableFeatureForDev' utility functions for the admin page to first enable the required flag, and then proceed to perform the testing.`

**Comment by seanlip** [comment]
@kevintab95 are you still planning to submit this?

File: Launching-new-features.md (hunk 3 above)
Comment (kevintab95) [reply Launching-new-features.md]: Done.

File: Launching-new-features.md (hunk 1 above)
Comment (kevintab95) [reply Launching-new-features.md]: Done.

File: Launching-new-features.md (hunk 2 above)
Comment (kevintab95) [reply Launching-new-features.md]: Added me for now.

**Review by kevintab95 — COMMENTED** [review]
@seanlip I've addressed comments. PTAL

**Comment by kevintab95** [comment]
> @kevintab95 are you still planning to submit this?

Yup, addressed comments now. Sorry for the delay!

**Review by seanlip — APPROVED** [review]
Looks good. Thanks!

**Pull request merged**